doc/doc-site/docs/off-chain/dss/changelog.md

```diff
@@ -13,11 +13,18 @@ Stable DSS builds are tracked on the [`dss-stable`] branch with tags like [`dss-
 1. Merge `main` into `dss-stable`.
 1. Push annotated tag to head of `dss-stable`.
 
-## [v2.1.0] (in progress)
+## [v2.1.0-rc1]
 
 ### Added
 
 - Price level events over MQTT (disabled by default, enable by adding `MQTT_PRICE_LEVELS=yes`, see `src/docker/example.env`) ([#753]).
+- Fix the `/tickers` `base_volume_nominal` field ([#749]).
+- Optimize the `/tickers` endpoint ([#731]).
```

Comment:
#731 is for hot upgrades

`/tickers` optimizations is #729

Reply:
Hot updgrade docs are not something related to the DSS version IMO.